scripts/chart-test.sh

```diff
@@ -81,3 +81,31 @@ if ! grep \""$PROMETHEUS_VER_HELM"\" pelorus-operator/bundle/metadata/properties
 else
   echo "OK: Prometheus version $PROMETHEUS_VER_HELM in sync with the pelorus-operator/bundle/metadata/properties.yaml"
 fi
+
+# Enforce operator version bump when charts folder is touched
+
+CURRENT_OPERATOR_VERSION="$(grep "VERSION ?= " pelorus-operator/Makefile  | cut -c 12-)"
+if ! git --no-pager diff --quiet $REMOTE/master --name-status charts/; then
+  OPERATOR_MAKEFILE_IN_MASTER="$(curl https://raw.githubusercontent.com/dora-metrics/pelorus/master/pelorus-operator/Makefile 2> /dev/null)"
+  OPERATOR_VERSION_IN_MASTER=$(echo "$OPERATOR_MAKEFILE_IN_MASTER" | grep "VERSION ?= " | cut -c 12-)
+  if [ "$OPERATOR_VERSION_IN_MASTER" == "$CURRENT_OPERATOR_VERSION" ]; then
+    echo "ERROR: Charts were modified, Operator needs version bumping!"
+    exit 1
+  fi
+fi
+
+# Check if both release candidate versions are the same
+# This also enforces that versions are both rc or both release
+# shellcheck disable=SC2207
+CURRENT_CHART_VERSION_ARRAY=( $(echo "$CURRENT_CHART_VERSION" | tr ' \-rc '  '  ') )
+CURRENT_CHART_VERSION_RC_VER=${CURRENT_CHART_VERSION_ARRAY[1]}
+# shellcheck disable=SC2207
+CURRENT_OPERATOR_VERSION_ARRAY=( $(echo "$CURRENT_OPERATOR_VERSION" | tr ' \-rc '  '  ') )
+CURRENT_OPERATOR_VERSION_RC_VER=${CURRENT_OPERATOR_VERSION_ARRAY[1]}
+if [ "$CURRENT_CHART_VERSION_RC_VER" != "$CURRENT_OPERATOR_VERSION_RC_VER" ]; then
+  echo "ERROR: Release candidate versions differs between charts (rc$CURRENT_CHART_VERSION_RC_VER) and operator (rc$CURRENT_OPERATOR_VERSION_RC_VER)!"
+  exit 1
+fi
+
+# TODO check for pending releases
```

Comment:
I think this TODO is wrong

we should automatically do a github release after a version without `rc` is merged

I think that is possible, just need to read the draft release body

do you agree with me?

Reply:
did not understand, can you give an example?

Reply:
Example. Create the release now regardless of any other conditions, based on the current state of the files and version that is taken from the source or passed while creating tag. Then we can have action that is creating tag as a link between closing PR and releasing new version of operator.

Reply:
I am going to open a separate issue for this, already worked too much in this PR